frontend/src/app/modules/common/back-routing/back-button.component.sass

```diff
@@ -0,0 +1,19 @@
+.op-back-button
+  padding: 0
+  line-height: 32px
+  margin: 0 1rem 0 0
+  width: 75px
+  height: 34px
+
+  &--icon
+    font-size: 1rem
+    line-height: 22px
+
+@media only screen and (max-width: 679px)
```

Comment:
You can actually indent this once and remove the class selector below; scss knows what you want

```
.op-back-button
  @media only screen and (max-width: 679px)
```

This works and is a really nice, bemmy-way to do media queries imo